addons/account/models/account_account.py

```diff
@@ -460,25 +466,40 @@ def _compute_opening_debit_credit(self):
 
     @api.depends('code')
     def _compute_account_type(self):
-        """ Compute the account type based on the account code.
-        Search for the closest parent account code and sets the account type according to the parent.
-        If there is no parent (e.g. the account code is lower than any other existing account code),
-        the account type will be set to 'asset_current'.
+        """
+            Compute the account type based on the account code.
+            Search for the closest parent account code and sets the account type according to the parent.
+            If there is no parent (e.g. the account code is lower than any other existing account code),
+            the account type will be set to 'asset_current'.
         """
         accounts_to_process = self.filtered(lambda r: r.code and not r.account_type)
+        self._helper_closest_parent_account(accounts_to_process, 'account_type', 'asset_current')
+
+    @api.depends('code')
+    def _compute_account_tags(self):
+        """
+            Compute the account tags based on the account code.
+            Search for the closest parent account code and sets the account tags according to the parent.
+            If there is no parent (e.g. the account code is lower than any other existing account code),
+            the account tags will be an empty list.
```

Comment:
IMHO, the docstring is a duplication of what the code does. The code is pretty clear, even for the `_compute_account_type`. If you follow my suggestion of adding named params in the line 487. I think we can remove the whole docstring for both computes and have a generic one on the helper function explaining in few words what we do.